core.js

```diff
@@ -1685,11 +1592,24 @@ exokit.setArgs = newArgs => {
 exokit.setVersion = newVersion => {
   version = newVersion;
 };
+
+/**
+ * Initialize classes and modules that require native bindings.
+ * Required before creating any windows or documents.
+ * Set rather than `require`d directly due to way `require` works with multithreading
+ * (for `Worker`), use this route to make sure binaries only get initialized once.
+ *
+ * @param {string} nativeBindingsModule - Path to native bindings JS module.
+ */
 exokit.setNativeBindingsModule = nativeBindingsModule => {
   nativeBindings = true;
 
   const bindings = require(nativeBindingsModule);
 
+  // Set in binding module to be referenced from other modules.
+  for (let key in bindings) { BindingsModule[key] = bindings[key]; }
```

Comment:
Nit: could be `const`.

Reply:
Can it? It's an iterator variable.

Reply:
Pretty sure.

```
o = {lol: 'zol', root: 'toot'};
for (const k in o) {
  console.log('key', k);
}
```

The exception is `for-i` where you assign to the loop variable yourself.